README.md

```diff
@@ -15,19 +15,17 @@ When you make a change to `test.sh`, run `make rebuild-test-runner`.
 | -------------------------- |:-------------------------------------------:|
 | `make build-nginx`         | Builds the NGINX image                      |
 | `make rebuild-nginx`       | Re-builds the NGINX image                   |
-| `make build-test-runner`   | Builds the image that will run `test.sh`    |
-| `make rebuild-test-runner` | Re-builds the image that will run `test.sh` |
+| `make build-test-runner`   | Builds the images used by the test stack (uses Docker compose)   |
+| `make rebuild-test-runner` | Re-builds the images used by the test stack |
 | `make start-nginx`         | Starts the NGINX container                  |
 | `make stop-nginx`          | Stops the NGINX container                   |
-| `make test`                | Runs `test.sh` against the NGINX container  |
+| `make test`                | Runs `test.sh` against the NGINX container (uses Docker compose)  |
```

Comment:
Nitpick: can you align the right-side `|`s?

Reply:
@JoshMcCullough  I also meant to ask, what is the purpose of copying the file artefacts out of the containers into the local dir in start-nginx?  I guess I wouldn't expect that to be part of a job which ostensibly only starts nginx.

Reply:
Viewing in a plain text editor, you'd want the columns to align. But like I said it's a nitpick.

Reply:
We copy the files out so they can be packaged up or otherwise used externally. We don't make use of the containers other than for building and testing the binary.

Reply:
Never mind, you need a running container to copy a file, duh :)  I'm moving the copy of ngx_http_auth_jwt_module.so into the build jobs.

Reply:
Ok then, never mind on that last statement as well. :)  I'll leave the cp of the module file itself, but will remove the other files as they're now part of OS packages.

Reply:
I've updated the readme and fixed the make failure with the commit I've just pushed.